test/utils/ignoreCSS.js

```diff
@@ -0,0 +1,6 @@
+function ignore() {
```

Comment:
Would do this in babel, to centralize all code transformation

```diff
diff --git a/babel.config.js b/babel.config.js
index e8bb547e7..09c6e06d3 100644
--- a/babel.config.js
+++ b/babel.config.js
@@ -128,6 +128,7 @@ module.exports = function getBabelConfig(api) {
       },
       'replace-date-fns-jalali-imports',
     ]);
+    plugins.push(['babel-plugin-transform-remove-imports', { test: '\\.css$' }]);
   }
 
   if (process.env.NODE_ENV === 'production') {
```

Reply:
Is the babel config the right place though? The import line needs to stay in the published package, we only want it gone in the jsdom tests.